app/controllers/api/series_controller.rb

```diff
@@ -5,4 +5,11 @@ class Api::SeriesController < Api::BaseController
   api_versions :v1
 
   filter_resources_by :account_id
+
+  filter_params :v4
```

Comment:
filter makes sense to me since we have the API supporting beta which shows both v3 and v4. sensible.